Jenkinsfile

```diff
@@ -24,40 +24,11 @@ node ('rocmtest')
     deleteDir( )
```

Comment:
You should probably remove this, so that cloning the entire repo doesn't happen on every build. Then you will need to remove the build directory before building, so the build artifacts won't cause problems.